Blockzilla/AppDelegate.swift

```diff
@@ -414,12 +415,29 @@ extension AppDelegate {
     }
 
     func setupExperimentation() {
+        let isFirstRun = !UserDefaults.standard.bool(forKey: OnboardingConstants.onboardingDidAppear)
```

Comment:
I didn't quite understand how first run was decided, but [this user defaults][0] seemed to be recording after onboarding was dismissed.

[0]: https://github.com/mozilla-mobile/focus-ios/blob/a477f1211b177380095728adc12f19fc922ca381/Blockzilla/BrowserViewController.swift#L376